sdk/monitor/azure-monitor-opentelemetry-autoconfigure/CHANGELOG.md

```diff
@@ -1,19 +1,16 @@
 # Release History
 
-## 1.4.0-beta.1 (Unreleased)
+## 1.4.0 (2025-09-19)
 
 ### Features Added
 
 - Added mapping for `enduser.pseudo.id` attribute to `user_Id` ([#46506](https://github.com/Azure/azure-sdk-for-java/pull/46506)
+- Add de-duping flag for standard/otel metrics & support for conditionally sending otel metrics to Breeze ([#46680](https://github.com/Azure/azure-sdk-for-java/pull/46680)
```

Comment:
This entry doesn't have any significance and should be avoided, as it's only for internal implementation.